From: Baolin Wang <baolin.wang@linux.alibaba.com>
Subject: mm: shmem: drop the unnecessary folio_nr_pages()
Date: Tue, 26 Aug 2025 17:35:55 +0800

We've got the number of pages in the folio earlier, thus remove the redundant
folio_nr_pages() call.

Link: https://lkml.kernel.org/r/67c80182ebd949e3894908e01e224697c143aabb.1756200587.git.baolin.wang@linux.alibaba.com
Signed-off-by: Baolin Wang <baolin.wang@linux.alibaba.com>
Cc: Hugh Dickins <hughd@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/shmem.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/shmem.c~mm-shmem-drop-the-unnecessary-folio_nr_pages
+++ a/mm/shmem.c
@@ -2430,7 +2430,6 @@ static int shmem_swapin_folio(struct ino
 		goto failed;
 	}
 	folio_wait_writeback(folio);
-	nr_pages = folio_nr_pages(folio);
 
 	/*
 	 * Some architectures may have to restore extra metadata to the
_
